src/static/css/main-content.less

```diff
@@ -3,9 +3,9 @@
    ========================================================================== */
 
 .section-image {
-    width: 65%;
-    margin: 0 auto;
-    padding-bottom: 1.4em;
+  width: 65%;
```

Comment:
Spacing in this file has all been converted from tabs to spaces. Use the link in the OP to preview without whitespace changes.